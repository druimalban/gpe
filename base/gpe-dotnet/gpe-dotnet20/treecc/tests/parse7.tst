// literal code definition sections
%{ xyz %}
%decls %{ xyz %}
%both %{ xyz %}
%end %{ xyz %}
%decls %end %{ xyz %}
%decls %both %{ xyz %}
%% xyz
