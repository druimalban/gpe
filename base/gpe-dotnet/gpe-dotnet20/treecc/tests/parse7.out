line 2: %literal 1  xyz 
line 3: %literal 2  xyz 
line 4: %literal 3  xyz 
line 5: %literal 5  xyz 
line 6: %literal 6  xyz 
line 7: %literal 3  xyz 
line 8: %literal 5  xyz

